dbt/include/mindsdb/macros/materialization/table.sql

```diff
@@ -14,17 +13,18 @@
 
   -- path
   {% for item in identifier.split('.') -%}
-     {{ target_relation_list.append('`{}`'.format(item))  }}
+    {{ target_relation_list.append('`{}`'.format(item))  }}
   {%- endfor %}
 
+  -- final
   {% set target_relation = target_relation_list | join('.') %}
 
   -- ... setup database ...
   -- ... run pre-hooks...
 
   -- build model
   {% call statement('main') %}
-       {{ apply_predictor_wrap(sql, predictor_name, target_relation) }}
+       {{ apply_predictor_wrap(sql, target_relation) }}
```

Comment:
Maybe it is worth to rename function to save_to_table_wrap

Reply:
no problem, will do that

Reply:
done